.eslintrc.json

```diff
@@ -4,11 +4,7 @@
     "browser": true,
     "es6": true
   },
-  "extends": [
-    "plugin:react/recommended",
-    "plugin:react-hooks/recommended",
-    "plugin:prettier/recommended"
```

Comment:
I assume this is now in the formatter?

Reply:
Correct, see https://github.com/trunk-io/configs#1-turn-off-formatting-errors-use-autoformatters-instead